artemis/src/main/java/tech/pegasys/artemis/services/ServiceController.java

```diff
@@ -59,5 +65,6 @@ public static void stopAll(CommandLineArguments cliArgs) {
     beaconChainService.stop();
     powchainExecuterService.shutdown();
     powchainService.stop();
+    p2pService.stop();
```

Comment:
Should p2pExecutorService.shutdown() also be called here?

Reply:
great catch.  thanks!